### ISSUE TITLE
Turn develop branch into master

### ISSUE THREAD

**Issue by matze** [christiangoltz/shaape#10]
Almost any project that I know of that is version controlled with Git, uses the master branch for development. Doing it the other way around, has some negative side effects, particularly here on GitHub:
- Actual development is not really visible and people might miss that the project is still alive.
- It's also kind of violating the principle of least astonishment.
- Making pull requests requires some additional work (setting up the upstream develop branch, merge correctly, yadayadayada).

So please, make master the main development branch. If you want to mark stable releases just tag and push _them_ as remote branches, e.g `stable/1.0.x`.

**Comment by christiangoltz** [comment]
Originally I started with the branching model from git flow (http://nvie.com/posts/a-successful-git-branching-model/), but I can see your point here. 
At the moment there is no reason for me to have a dedicated stable branch. I just tagged the v1.0.0 release on master and merged develop into master. Should be sufficient unless we need some hotfix, at which point I could still branch from the tag.
